### DISCUSSION TITLE
AttributeError: module 'discord.ext.commands' has no attribute 'has_premissions'

### DISCUSSION THREAD

**Discussion by Jaegerist** [Rapptz/discord.py#7283] [Q&A]
in github could not find an answer to this question. The fact is that the code is written off 100% from the guide on YouTube. The code works completely for the author of the video, but I get an error.

`@client.command(pass_context=True)`
`@commands.has_premissions(administrstor=True)`
`async def clear(ctx, amount=100):`
`await ctx.channel.purge(limit=amount)`

or

![image](https://user-images.githubusercontent.com/87984091/127033289-50626f7a-fb0e-48b5-9488-87a88f22fb25.png)

it is worth noting that `pass_context` does not play any role at all: what's with it, what's without it, there is no difference

Imports:
![image](https://user-images.githubusercontent.com/87984091/127033404-86580c34-58c3-4cc9-9d4a-fab21619848a.png)

Error:
![image](https://user-images.githubusercontent.com/87984091/127033498-3a38fd48-3a22-4be2-82fc-7adb00c60cb5.png)

**Comment by imvaskel** [comment]
Both permissions and administrator have been spelt wrong.

**Reply by Jaegerist** [reply]
lol, sorry

**Reply by helish88** [reply]
lol
